Error when cross-compiling for windows under linux with mingw32

Error: execution of an external compiler program 'i686-w64-mingw32-gcc -c -w -mno-ms-bitfields -DWIN32_LEAN_AND_MEAN  -I/usr/lib/nim -o /home/hao/linux_share/nim/PTA/nimcache/stdlib_system.o /home/hao/linux_share/nim/PTA/nimcache/stdlib_system.c' failed with exit code: 1

/home/hao/linux_share/nim/PTA/nimcache/stdlib_system.c:19:10: fatal error: Windows.h: No such file or directory
 #include "Windows.h"

--- a/lib/windows/winlean.nim
+++ b/lib/windows/winlean.nim
@@ -1101,7 +1101,7 @@ proc ConvertThreadToFiber*(param: pointer): pointer {.stdcall, discardable, dynl
 proc ConvertThreadToFiberEx*(param: pointer, flags: int32): pointer {.stdcall, discardable, dynlib: "kernel32", importc.}
 proc DeleteFiber*(fiber: pointer): void {.stdcall, discardable, dynlib: "kernel32", importc.}
 proc SwitchToFiber*(fiber: pointer): void {.stdcall, discardable, dynlib: "kernel32", importc.}
-proc GetCurrentFiber*(): pointer {.stdcall, importc, header: "Windows.h".}
+proc GetCurrentFiber*(): pointer {.stdcall, importc, header: "windows.h".}
 
 proc toFILETIME*(t: int64): FILETIME =
   ## Convert the Windows file time timestamp ``t`` to ``FILETIME``.

